src/common/Channel.cpp

```diff
@@ -202,6 +203,20 @@ void Channel::addMessagesAtStart(std::vector<MessagePtr> &_messages)
 {
     std::vector<MessagePtr> addedMessages =
         this->messages_.pushFront(_messages);
+    /*
+    if (getSettings()->similarityEnabled)
+    {
+        for (int i = 0; i < addedMessages.size(); ++i)
+        {
+            if (IrcMessageHandler::similarityRecentMessages(addedMessages, i) >
+                getSettings()->similarityPercent.getValue())
+            {
+                addedMessages[i]->flags.set(MessageFlag::Hidden, true);
+                addedMessages[i]->flags.set(MessageFlag::Disabled, true);
+            }
+        }
+    }
+    */
```

Comment:
Why is this uncommented

Reply:
https://github.com/Chatterino/chatterino2/pull/1496/files/e14be92deafb4aa3b59763361c0876ea7b4b6248#diff-be41aa477922fd78d2e9dac3f5ac1711R105

Reply:
so what should be done about this